packages/web-app-files/src/components/Spaces/SpaceContextActions.vue

```diff
@@ -68,89 +67,103 @@ export default defineComponent({
       required: true
     },
     items: {
-      type: Array,
+      type: Array as PropType<Resource[]>,
       required: true
     }
   },
-  setup() {
-    return {
-      maxQuota: useCapabilitySpacesMaxQuota()
-    }
-  },
-  computed: {
-    quotaModalSelectedSpaces() {
-      return [this.$data.$_editQuota_selectedSpace]
-    },
-    quotaModalIsOpen() {
-      return this.$data.$_editQuota_modalOpen
-    },
-    readmeContentModalIsOpen() {
-      return this.$data.$_editReadmeContent_modalOpen
-    },
-    menuSections() {
-      const sections = []
+  setup(props) {
+    const instance = getCurrentInstance().proxy as any
+    const router = useRouter()
+    const store = useStore()
+    const { space } = toRefs(props)
```

Comment:
So we can use space as a ref in the computed, otherwise we would need to use props.space, which would be fine as well. 
Just destructuring props loses reactivity